### PR TITLE
chore: Generate gRPC client, server code from mayastor.proto file and CR client code form CRD file

### PR DESCRIPTION
**Pull request by rohan2794**
generate gRPC server and clinet code from mayastor.proto file which is present in install bundle. And also generate CR client code from CRD file present in install bundle.
Replace existing gRPC and CR files with generated files.
Add arguments to e2e-test.sh file which will enable and disable gRPC and CR code generatiion.

JIRA: https://mayadata.atlassian.net/browse/MQ-2711

### CODE REVIEW

**Review by chriswldenyer — APPROVED** [review]
Looks reasonable, some suggestions.

File: scripts/e2e-test.sh [hunk 1]
```diff
@@ -139,6 +141,14 @@ while [ "$#" -gt 0 ]; do
       shift
       tag=$1
       ;;
+    -g|--grpc_code_gen)
```
[the remark this answers is not in the document]
Comment (rohan2794) [reply scripts/e2e-test.sh:150]: it's already there in help section

File: scripts/e2e-test.sh [hunk 2]
```diff
@@ -391,6 +401,30 @@ mkdir -p "$logsdir"
 
 test_failed=0
 
+# Generate gRPC server and client code from mayastor.proto file
+if [ -n "$grpc_code_gen" -a "$grpc_code_gen" = true ]; then
```
[the remark this answers is not in the document]
Comment (chriswldenyer) [reply scripts/e2e-test.sh]: Sorry, I meant you don't need the "-n $grpc_code_gen" part.

File: scripts/e2e-test.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (chriswldenyer) [reply scripts/e2e-test.sh:150]: Oops, sorry, not sure how I missed that...

**Review by chriswldenyer — APPROVED** [review]

**Review by payes — APPROVED** [review]

**Review by blaisedias — APPROVED** [review]
LGTM

**Pull request merged**